smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HttpBindingProtocolGenerator.kt

```diff
@@ -171,7 +170,7 @@ abstract class HttpBindingProtocolGenerator : ProtocolGenerator {
                         writer.addImport(SwiftDependency.CLIENT_RUNTIME.target)
 
                         if (shouldRenderCodingKeysForEncodable) {
-                            generateCodingKeysForMembers(ctx, writer, httpBodyMembers)
+                            codingKeysGenerator?.generateCodingKeysForMembers(ctx, writer, httpBodyMembers)
```

Comment:
Dynamic node decoding is removed; it was an implementation detail of the 3rd party XML decoding package.

`codingKeysGenerator` is now optional since not all protocols use coding keys.